ui/src/main/res/xml/app_restrictions.xml

```diff
@@ -9,5 +9,6 @@
         android:description="@string/disable_config_export_description"
         android:key="disable_config_export"
         android:restrictionType="bool"
-        android:title="@string/disable_config_export_title" />
+        android:title="@string/disable_config_export_title"
+		android:singleLineTitle="false" />
```

Comment:
I believe the Android Studio formatter would place these lines above `android:title`, can you please format both the changed files when you amend your commit?